File: reunet-api/src/main/java/com/reunet/app/models/Message.java
package com.reunet.app.models;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import java.util.Date;
import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Entity
@Table(name = "message")
public class Message {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "message_id")
    @JsonProperty("message_id")
    private Long messageId;

    @JsonProperty("user_id")
    @Column(name = "user_id")
    private Long userId;

    @JsonProperty("group_id")
    @Column(name = "group_id")
    private Long groupId;

    @Column(name = "message_body")
    @JsonProperty("body")
    private String messageBody;

    @JsonProperty("created_at")
    @Column(name = "created_at")
    private Date createdAt;
}

File: front_movil/app/src/main/java/com/reunet/app/models/Message.kt
package com.reunet.app.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Message(
    @SerializedName("message_id") var id: String,
    @SerializedName("user_id") var userId: String,
    @SerializedName("group_id") var groupId: String,
    @SerializedName("body") var body: String,
    @SerializedName("created_at") var createdAt: String
): Serializable